lib/error_tracker.ex

```diff
@@ -108,12 +108,14 @@ defmodule ErrorTracker do
     {kind, reason} = normalize_exception(exception, stacktrace)
     {:ok, stacktrace} = ErrorTracker.Stacktrace.new(stacktrace)
     {:ok, error} = Error.new(kind, reason, stacktrace)
-
     context = Map.merge(get_context(), given_context)
 
-    {_error, occurrence} = upsert_error!(error, stacktrace, context, reason)
+    ignorer = Application.get_env(:error_tracker, :ignorer)
 
-    occurrence
+    if ignorer && !ignorer.ignore?(error, context) do
```

Comment:
I think this does not work as we expect.

It should store the error if either no ignorer is set or it is set but it returns to not ignore the error.

Something like:
```suggestion
    if is_nil(ignorer) || !ignorer.ignore?(error, context) do
```

Reply:
You are right. Fixed!